### PR TITLE
capture.cpp: Replace WriteFile with <fstream>

### PR DESCRIPTION
**Pull request by glebm**

### CODE REVIEW

**Comment by AJenbo** [comment]
@glebm nice this will solve #537 (only blocker for 1.0.1) and takes out some uglyness from the code. But currently file_util fails on Windows and Mac.

File: SourceS/file_util.h [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+inline bool FileExists(const char *path)
+{
+#if _POSIX_C_SOURCE >= 200809L
+	return access(path, F_OK) == 0;
+#else
+	auto *file = std::fopen(name.c_str(), "rb");
+	if (file) {
+		fclose(file);
+		return true;
+	} else {
```
Comment (AJenbo) [inline SourceS/file_util.h]: I would prefer and early return instead:
```
	if (!file) {
		return false;
	}
	fclose(file);
	return true;
```

File: SourceS/file_util.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#include <unistd.h>
+#else
+#include <cstdio>
+#endif
+
+namespace dvl {
+
+inline bool FileExists(const char *path)
+{
+#if _POSIX_C_SOURCE >= 200809L
```
Comment (AJenbo) [inline SourceS/file_util.h]: I think access is already in 200112L
```suggestion
#if _POSIX_C_SOURCE >= 200112L

```

Comment (glebm) [reply SourceS/file_util.h]: Done

File: SourceS/file_util.h (hunk 1 above)
Comment (glebm) [reply SourceS/file_util.h]: Done

**Pull request merged**